src/tests/conformance/query/peer_shard_info.rs

```diff
@@ -96,7 +90,7 @@ async fn check_relay_for_key_type(key_type: u8, relays: u32) {
 
     // Create a dummy key with the specified key type.
     let mut key = vec![key_type]; // Place the key type as the first byte.
-    key.resize(33, 0x1); // Append 32 bytes serving as a dummy public key.
+    key.resize(PUBLIC_KEY_LENGTH + 1, 0x1); // Append 32 bytes serving as a dummy public key.
```

Comment:
```suggestion
    key.resize(PUBLIC_KEY_LENGTH, 0x1); // Append 32 bytes serving as a dummy public key.
```

Reply:
that, or the `const` should be `32B`

Reply:
Thanks. It made one test fail, but I didn't run it after changing this :-/

Reply:
Done